client/src/Chat.jsx

```diff
@@ -1,44 +1,148 @@
-        messages{
-          content
-          id
-          user
-        }
-      }`
-
-const Messages = ({user}) => {
-  const {data} = useQuery(GET_MESSAGES);
-  if(!data){
-    return null
+const POST_MESSAGES = gql`
+  mutation ($user: String!, $content: String!) {
+    postMessage(user: $user, content: $content)
   }
+`;
 
-  return JSON.stringify(data)
-}
-  const Chat = () => {
-      return(
-          <div><Messages user="Olasumbo"/></div>
-      )
+const Messages = ({ user }) => {
+  const { data } = useQuery(GET_MESSAGES);
+  if (!data) {
+    return null;
   }
-  
 
-  export default () =>(
-    <ApolloProvider client={client}>
-        <Chat />
-    </ApolloProvider>
-  );
\ No newline at end of file
+  return (
+    <>
+      {data.messages.map(({ id, user: messageUser, content }) => (
+        <div
+          style={{
+            display: 'flex',
+            justifyContent: user === messageUser ? 'flex-end' : 'flex-start',
+            paddingBottom: '1em'
+          }}
+        >
+          {user !== messageUser && (
+            <div
+              style={{
+                height: 50,
+                width: 50,
+                marginRight: '0.5em',
+                border: '2px solid #e5e6ea',
+                borderRadius: 25,
+                textAlign: 'center',
+                fontSize: '18pt',
+                paddingTop: 5
+              }}
+            >
+              {messageUser.slice(0, 2).toUpperCase()}
+            </div>
+          )}
+
+          <div
+            style={{
+              background: user === messageUser ? '#58bf56' : '#e5e6ea',
+              color: user === messageUser ? 'white' : '#black',
+              padding: '1em',
+              borderRadius: '1em',
+              maxWidth: '60%'
+            }}
+          >
+            {content}
+          </div>
+        </div>
+      ))}
+    </>
+  );
+};
+const Chat = () => {
+  const [state, stateSet] = React.useState({
+    user: 'Olasumbo',
+    content: ''
+  });
+
+  const [postMessage] = useMutation(POST_MESSAGES);
+
+  const onSend = () => {
+    if (state.content.length > 0) {
+      postMessage({
+        variables: state
+      });
+    }
+    stateSet({
+      ...state,
+      content: ''
+    });
+  };
+  return (
+    <Container>
+      <Messages user={state.user} />
+      <Row>
+        <Col xs={2} style={{ padding: 0 }}>
+          <FormInput
+            label="User"
+            value={state.user}
+            onChange={(evt) =>
+              stateSet({
+                ...state,
+                user: evt.target.value
+              })
+            }
+          />
+        </Col>
+        <Col xs={8}>
+          <FormInput
+            label="Content"
+            value={state.content}
+            onChange={(evt) =>
+              stateSet({
+                ...state,
+                content: evt.target.value
+              })
+            }
+            onKeyUp={(evt) => {
+              if (evt.keyCode === 13) {
```

Comment:
Magin number antipattern. Create a variable called `ENTER_KEY_CODE` or smth like that and assign to it 13 and use it